src/Examples/Component Examples/Examples.ContactBook/Views.fs

```diff
@@ -26,71 +26,71 @@ module CustomHooks =
-                                | Choice2Of2 exn -> state.Set (Deferred.Failed exn)
-                            }
-                        )
+                        | _ ->
+                            ()
+                    ),
+                    triggers = [ EffectTrigger.AfterInit ]
+                )
 
-                    | _ ->
-                        ()
-                ),
-                triggers = [ EffectTrigger.AfterInit ],
-                callerLineNumber = identity
+                state
             )
 
-            state
 
     type IComponentContext with
 
         member inline ctx.useDeferred<'a>(outer: IWritable<'a>, delay: int, [<CallerLineNumber>] ?lineNumber: int) =
-            let inner = ctx.useState (outer.Current, identity = $"line: {lineNumber} - inner state")
+            ctx.useGrouped (lineNumber.Value, fun ctx ->
+                let inner = ctx.useState (outer.Current)
+
+                ctx.useEffect (
+                    handler = (fun _ ->
+                        inner.Set outer.Current
+                    ),
+                    triggers = [ EffectTrigger.AfterChange outer ]
+                )
 
-            ctx.useEffect (
-                handler = (fun _ ->
-                    inner.Set outer.Current
-                ),
-                triggers = [ EffectTrigger.AfterChange outer ],
-                identity = $"line: {lineNumber} - outer sync effect"
-            )
+                let timer = ctx.useState (
+                    initialValue = (
+                        let timer = new Timer(Interval = delay, AutoReset = false)
+                        timer.Elapsed.Add (fun _ ->
+                            outer.Set inner.Current
+                        )
+                        timer
+                    ),
+                    renderOnChange = false
+                )
 
-            let timer = ctx.useState (
-                initialValue = (
-                    let timer = new Timer(Interval = delay, AutoReset = false)
-                    timer.Elapsed.Add (fun _ ->
-                        outer.Set inner.Current
-                    )
-                    timer
-                ),
-                renderOnChange = false,
-                identity = $"line: {lineNumber} - timer state"
-            )
+                ctx.useEffect (
+                    handler = (fun _ ->
+                        timer.Current.Stop()
+                        timer.Current.Start()
+                    ),
+                    triggers = [ EffectTrigger.AfterChange inner ]
+                )
 
-            ctx.useEffect (
-                handler = (fun _ ->
-                    timer.Current.Stop()
-                    timer.Current.Start()
-                ),
-                triggers = [ EffectTrigger.AfterChange inner ],
-                identity = $"line: {lineNumber} - inner sync effect"
+                inner
```

Comment:
`ctx.useDeferred` with `useGrouped`

(MISC: useDeferred is a really bad name here. It should be named `debounce` or something)